src/quo2/components/buttons/slide_button/utils.cljs

```diff
@@ -0,0 +1,38 @@
+(ns quo2.components.buttons.slide-button.utils
+  (:require
+    [quo2.components.buttons.slide-button.constants :as constants]
+    [quo2.foundations.colors :as colors]))
+
+;; Utils
+(defn slider-color
+  "- `color-key`               `:main`/`:track`
+   - `customization-color` Customization color"
+  [color-key customization-color]
+  (let [colors-by-key {:main  (colors/custom-color-by-theme customization-color 50 60)
+                       :track (colors/custom-color-by-theme customization-color 50 60 10 10)}]
+    (color-key colors-by-key)))
+
+(defn clamp-value
+  [value min-value max-value]
+  (cond
+    (< value min-value) min-value
+    (> value max-value) max-value
+    :else               value))
+
+(defn calc-usable-track
+  "Calculate the track section in which the
+  thumb can move in. Mostly used for interpolations."
+  [track-width thumb-size]
+  (- track-width (* constants/track-padding 2) thumb-size))
```

Comment:
imo this could be more readable if you use the `->` or `->>`  functions here or alternatively using a let var would help too:
```
[track-width thumb-size]
(let [double-padding (* constants/track-padding 2)]
  (- track-width double-padding thumb-size))
```

Reply:
done